taxcalc/functions.py

```diff
@@ -1123,14 +1123,14 @@ def F2441(MARS, earned_p, earned_s, f2441, CDCC_c, e32800,
 
 @jit(nopython=True)
 def EITCamount(phasein_rate, earnings, max_amount,
-               phaseout_start, eitc_agi, phaseout_rate):
+               phaseout_start, agi, phaseout_rate):
```

Comment:
Would it be clearer to use `c00100` here instead of introducing a new variable name of `agi`, which might be interpreted as a different concept? functions.py currently only uses the name `c00100` to refer to AGI in code.